legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -390,8 +390,20 @@ class FilingTitles(str, Enum):
         'name': 'courtOrder',
         'title': 'Court Order',
         'displayName': 'Court Order',
-        'code': 'NOFEE'
-    },
+        'codes': {
+                'SP': 'COURT',
+                'GP': 'COURT',
+                'CP': 'COURT',
+                'BC': 'COURT',
+                'BEN': 'COURT',
+                'CC': 'COURT',
+                'ULC': 'COURT',
+                'C': 'COURT',
+                'CBEN': 'COURT',
+                'CUL': 'COURT',
+                'CCC': 'COURT',
+            }
```

Comment:
Please confirm with Mihai that these pay codes are configured in Dev, Test and Prod.

After merging, please test some court orders and verify the receipt, please.